x-pack/plugins/security_solution/cypress/e2e/detection_alerts/alert_table_controls.cy.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewRule } from '../../objects/rule';
+import { FIELDS_BROWSER_BTN } from '../../screens/rule_details';
+import {
+  DATA_GRID_COLUMN_ORDER_BTN,
+  DATA_GRID_FIELDS,
+  DATA_GRID_FULL_SCREEN,
+  GET_DATA_GRID_HEADER,
+  GET_DATA_GRID_HEADER_CELL_ACTION_GROUP,
+} from '../../screens/shared';
+import { createCustomRuleEnabled } from '../../tasks/api_calls/rules';
+import { cleanKibana } from '../../tasks/common';
+import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
+import { login, visit } from '../../tasks/login';
+import { ALERTS_URL } from '../../urls/navigation';
+
+describe('Alert Table Contorls', { testIsolation: false }, () => {
+  before(() => {
+    cleanKibana();
+    login();
+    createCustomRuleEnabled(getNewRule());
+    visit(ALERTS_URL);
+    waitForAlertsToPopulate();
+  });
+
+  it('full screen', () => {
+    cy.get(DATA_GRID_FULL_SCREEN)
+      .should('have.attr', 'aria-label', 'Enter fullscreen')
+      .trigger('click')
+      .should('have.attr', 'aria-label', 'Exit fullscreen')
+      .trigger('click');
```

Comment:
this trigger here is not useful,  you're not testing anything after clicking

Reply:
@PhilippeOberti , if you see the describe clause, there is a option called `testIsolation: false` which makes sure that the page does not refresh after every test so as to save time. Since, page does not refresh, test has to cleanup after itself.

Also, since all tests are different, cleanup cannot same and in `afterEach` clause.

Reply:
ok that makes sense! I wonder if it would make it more readable to extract this into a method like `exitFullScreen()`. But if we only use it once here, not sure its valuable...